pkg/registry/networking/networkpolicy/strategy_test.go

```diff
@@ -0,0 +1,319 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
nit: 2021

Reply:
well, the first commit of this was 2020! haha! but let me change this

Reply:
I do wonder if this date should be based on the commit timestamp or when the PR is merged :thinking:

Reply:
yah, no problem at all :D will change now so CI have time to run again still today :)

Reply:
changed :D